bin/setup.js

```diff
@@ -0,0 +1,108 @@
+const spaceImport = require("contentful-import");
+const exportFile = require("../contentful/export.json");
+const inquirer = require("inquirer");
+const chalk = require("chalk");
+const path = require("path");
+const { writeFileSync } = require("fs");
+
+const argv = require("yargs-parser")(process.argv.slice(2));
+
+console.log(`
+  To set up this project you need to provide your Space ID
+  and the belonging API access tokens. Please use an empty space for this.
+  You can find all the needed information in your Contentful space under:
+  ${chalk.yellow(
+    `app.contentful.com ${chalk.red("->")} Space Settings ${chalk.red(
+      "->"
+    )} API keys`
+  )}
+  The ${chalk.green("Content Management API Token")}
+    will be used to import and write data to your space.
+  The ${chalk.green("Content Delivery API Token")}
+    will be used to ship published production-ready content in your Gatsby app.
+  The ${chalk.green("Content Preview API Token")}
+    will be used to show not published data in your development environment.
+  Ready? Let's do it! 🎉
+`);
+
+const questions = [
+  {
+    name: "spaceId",
+    message: "Your Space ID",
+    when: !argv.spaceId && !process.env.CONTENTFUL_SPACE_ID,
+    validate: (input) =>
+      /^[a-z0-9]{12}$/.test(input) ||
+      "Space ID must be 12 lowercase characters",
+  },
+  {
+    name: "managementToken",
+    when: !argv.managementToken,
+    message: "Your Content Management API access token",
+  },
+  {
+    name: "accessToken",
+    when:
+      !argv.accessToken &&
+      !process.env.CONTENTFUL_ACCESS_TOKEN &&
+      !argv.deliveryToken &&
+      !process.env.CONTENTFUL_DELIVERY_TOKEN,
+    message: "Your Content Delivery API access token",
+  },
+];
+
+inquirer
+  .prompt(questions)
+  .then(({ spaceId, managementToken, accessToken }) => {
+    const {
+      CONTENTFUL_SPACE_ID,
+      CONTENTFUL_ACCESS_TOKEN,
+      CONTENTFUL_DELIVERY_TOKEN,
+    } = process.env;
+
+    // env vars are given precedence followed by args provided to the setup
+    // followed by input given to prompts displayed by the setup script
+    spaceId = CONTENTFUL_SPACE_ID || argv.spaceId || spaceId;
+    managementToken = argv.managementToken || managementToken;
+    // Some scripts that set up this repo use `deliveryToken` and
+    // `CONTENTFUL_DELIVERY_TOKEN`, instead of `accessToken` and
+    // `CONTENTFUL_ACCESS_TOKEN`. Until all scripts are updated to
+    // use `accessToken` and `CONTENTFUL_ACCESS_TOKEN` both variations
+    // will work.
+    accessToken =
+      CONTENTFUL_ACCESS_TOKEN ||
+      CONTENTFUL_DELIVERY_TOKEN ||
+      argv.accessToken ||
+      argv.deliveryToken ||
+      accessToken;
+
+    console.log("Writing config file...");
+    const configFiles = [`.env.development`, `.env.production`].map((file) =>
```

Comment:
We will only be using `.env` files in development so no need to add a postfix. Change this to just `.env`.